db.js

```diff
@@ -226,20 +227,16 @@ exports.init = function (dir, config) {
       */
     }
 
-    for (var indexName in indexes) {
+    for (const indexName in indexes) {
       result.indexes[indexName] = indexes[indexName].seq.value
     }
 
     return result
   }
 
   function clearIndexes() {
-    for (var index in indexes)
-      indexes[index].remove(() => {})
-  }
-
-  var indexes = {
-    base: baseIndex
+    for (const indexName in indexes)
+      indexes[indexName].remove(() => {})
```

Comment:
The diff above looks weird, here's what's actually going on:

- `var indexes = {....` moved to the top of this file
- `for (var index....` changed to `for (const indexName....`